frontend/packages/noobaa-storage-plugin/src/components/capacity-breakdown/capacity-breakdown-card.tsx

```diff
@@ -51,7 +93,8 @@ const BreakdownCard: React.FC<DashboardItemProps> = ({
   const ind = top5MetricsStats.findIndex((v) => v.name === 'Others');
   if (ind !== -1) {
     top5MetricsStats[ind].name = 'Cluster-wide';
-    top5MetricsStats[ind].link = '';
+    top5MetricsStats[ind].link =
+      'Any NON Object bucket claims that were created via an S3 client or via the NooBaa UI system.';
```

Comment:
```suggestion
      'Object Bucket Claims created via an S3 client or through the NooBaa UI System.';
```
@shirimordechay could you suggest what should be written here?

Reply:
@bipuladh this has to be this way only.
FYI the text comes from design team.

Reply:
I have had texts changed later and this doesn't look correct to me(both semantics and grammar). Please ask them to verify it.

Reply:
Thanks but what you are suggesting doesn't look right.